spec/arns_spec.lua

```diff
@@ -516,23 +516,25 @@ describe("arns", function()
 				purchaseType = "lease",
 				years = 2,
 				name = "test-name",
+				currentTimestamp = timestamp,
 			}
 			_G.DemandFactor.currentDemandFactor = demandFactor
 			assert.are.equal(expectedCost, arns.getTokenCost(intendedAction))
 		end)
-		it("should return the correct token cost for a permabuy", function()
+		it("should return the correct token cost for a buying permanently owned", function()
```

Comment:
funny grammar

Reply:
addressed in [da8ab39](https://github.com/ar-io/ar-io-network-process/pull/156/commits/da8ab39db7266c47d5652789cbbb17c4af73464b) and [a2bf822](https://github.com/ar-io/ar-io-network-process/pull/156/commits/a2bf8227d1ca5341b12c161c34eaf7ab35628ab2)